examples/02_decoding/plot_haxby_anova_svm.py

```diff
@@ -35,7 +35,7 @@
 
 # We now have 2 conditions
 print(conditions.unique())
-session = behavioral[condition_mask]
+session = behavioral[condition_mask].to_records(index=False)
```

Comment:
Could you add a comment above explaining this. For non pandas experts, it will not be obvious what's happening there.

Reply:
+1

Reply:
I added a comment and print statement to try and better clarify this. If there's something else you would prefer (or think there's a better way it could be worded), happy to update !

Reply:
It's definitely useful.

I must admit that I never heard about the "to_record" method in pandas, and that I don't have a good feeling between the difference between its output and a pandas dataframe.

Reply:
Iterating through the dataframe collapses the entries into tuples so their structure isn't preserved by `np.unique` (unless you specify an axis), but that would involve modifying the sklearn function, which I didn't think this was worth !